client/db/bolt/db.go

```diff
@@ -299,6 +300,65 @@ func (db *BoltDB) CreateAccount(ai *dexdb.AccountInfo) error {
 	})
 }
 
+// DeleteAccount removes the account by host.
+func (db *BoltDB) DeleteAccount(ai *dexdb.AccountInfo) error {
+	acctKey := []byte(ai.Host)
+	return db.acctsUpdate(func(accts *bbolt.Bucket) error {
+		return accts.DeleteBucket(acctKey)
+	})
+}
+
+// DisableAccount copies the AccountInfo to disabledAccounts and deletes
+// the AccountInfo from accounts.
+func (db *BoltDB) DisableAccount(ai *dexdb.AccountInfo) error {
+	// Copy AccountInfo to disabledAccounts
+	err := db.acctsDisable(func(disabledAccounts *bbolt.Bucket) error {
+		acct, err := disabledAccounts.CreateBucket(ai.EncKey)
+		if err != nil {
+			return fmt.Errorf("failed to create disabledAccount bucket")
+		}
+		err = acct.Put(accountKey, ai.Encode())
+		if err != nil {
+			return fmt.Errorf("accountKey put error: %v", err)
+		}
+		return err
+	})
+	if err != nil {
+		return err
+	}
+	err = db.DeleteAccount(ai)
+	if err != nil {
+		if err == bbolt.ErrBucketNotFound {
+			log.Warnf("failed to to delete account: %s err: %v", ai.Host, err)
+		} else {
+			return err
+		}
+	}
+	return nil
+}
+
+// DisabledAccount gets the AccountInfo from disabledAccount associated with
+// the specified EncKey.
+func (db *BoltDB) DisabledAccount(encKey []byte) (*dexdb.AccountInfo, error) {
```

Comment:
I also don't think this needs to be exported.

Reply:
This function is not used in any code. I needed to retrieve the Disabled Accounts to verify that the DisableAccount function worked in a unit test. It seemed like a useful function to have if we do the reinstate work so I went ahead and added the function. I'll remove the function and add the capability to the unit test if that is preferable.

Reply:
I think it's ok to leave, but should still be lower case.